app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,37 @@
+.centered {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.form-container {
+  margin: 0px auto;
+  border: 2px solid rgba(59, 59, 145, 0.705);
+  padding: 20px;
+  border-radius: 5px;
+  width: 40%;
```

Comment:
this won't look good on a phone I think 🤔 since it'll take 40% of the screen and that's way too narrow. WDYT if we just give it a maximum width? I believe `max-width: 400px` would work fine, this way it will never expand beyond 400px but it will take all the screen space it needs on a phone.